spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/services/StatusUpdater.java

```diff
@@ -74,6 +74,8 @@ public StatusUpdater(InstanceRepository repository, InstanceWebClient instanceWe
 		log.debug("Update status for {}", instance);
 		return this.instanceWebClient.instance(instance).get().uri(Endpoint.HEALTH)
 				.exchangeToMono(this::convertStatusInfo).log(log.getName(), Level.FINEST)
+				.retryWhen(Retry.max(5)
```

Comment:
Can this "hide" real outages/restarts?
-> service is down for a short moment, but due to the retries we don not see this in event log
-> can we differentiate between network errors and service down?